### PR TITLE
check isDrag to not trigger dragging code onclick.

### PR DESCRIPTION
**Pull request by diegovar**
Currently clicking an element triggers a lot of onDragEnd logic when clicking elements in the canvas. Check if isDrag=false to skip that when it's just a click.

### CODE REVIEW

**Comment by swissspidy** [comment]
Which issue does this address?

**Comment by diegovar** [comment]
> Which issue does this address?

None in particular. I was doing some research due to what Paul said that moving things in the canvas is very slow and laggy, and found that clicking any element in the canvas triggers the entire dragEnd logic, which in some cases (eg: video) is very slow and janky. This tiny PR makes selecting elements in the canvas a lot faster and less janky, and I guess could be filed under misc performance improvements.

File: assets/src/edit-story/components/canvas/singleSelectionMoveable.js [hunk 1]
```diff
@@ -271,7 +271,10 @@ function SingleSelectionMoveable({
         set(frame.translate);
         return undefined;
       }}
-      onDragEnd={({ target }) => {
+      onDragEnd={({ isDrag, target }) => {
```
Comment (miina) [inline assets/src/edit-story/components/canvas/singleSelectionMoveable.js:274]: Should this go for the `multiSelectionMoveable` as well?

Comment (diegovar) [reply assets/src/edit-story/components/canvas/singleSelectionMoveable.js:274]: Done.

Comment (diegovar) [reply assets/src/edit-story/components/canvas/singleSelectionMoveable.js:274]: Actually had to undo that. `onDragGroupEnd` calls `clickHandled` which allows selecting one element in a group, so in this case (multi select) we shouldn't check for isDrag.

**Review by miina — APPROVED** [review]

**Pull request merged**